src/main/java/com/example/bookstore/service/user/UserServiceImpl.java

```diff
@@ -22,6 +27,14 @@ public UserResponseDto register(UserRegistrationRequestDto requestDto) {
                     requestDto.getEmail()));
         }
         User user = userMapper.toModel(requestDto);
+        Set<Role> roles = new HashSet<>();
```

Comment:
Let's stick to the task description
As a User, I want to register, so I can place orders.

So, during the registration user getting role USER. Admin is redundant here